### DISCUSSION TITLE
blog/2023/230114-matrix-qq-wechat-bridge/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#7] [Announcements]
# blog/2023/230114-matrix-qq-wechat-bridge/

QQ 与微信客户端在各个平台上均存在不同程度上的作恶行为，包括但不限于：后台常驻保活、占用大量存储空间、远程聊天记录管理不善等，与之形成强烈反差的是 Telegram 客户端。我们急切需要一个像 Telegram 一样友好的工具来代理接收和发送 QQ 和微信消息。

https://blog.arisa.moe/blog/2023/230114-matrix-qq-wechat-bridge/

**Comment by ANGJustinl** [comment]
您有没有考虑过直接装饰gocqhttp之类，所有的消息获取和处理均依赖gocq自带的api :S
目前看过的类似是这个没有缓存的网页版
https://github.com/Stapxs/Stapxs-QQ-Lite-2.0

**Reply by zhanbao2000** [reply]
看起来觉得很不错，可以考虑近期尝试一下

**Reply by ANGJustinl** [reply]
dalao加油，不过gocq确实高效啊，我自己挂2个加起来100个群还能稳定在200mram左右，而且现在还有获取收藏表情包的api
